verl/workers/fsdp_workers.py

```diff
@@ -643,45 +687,59 @@ def generate_sequences(self, prompts: DataProto):
         return output
 
     @register(dispatch_mode=Dispatch.DP_COMPUTE_PROTO)
-    def compute_log_prob(self, data: DataProto):
+    def compute_log_prob(self, data: DataProto, no_lora=False):
```

Comment:
It's not appropriate to add a separate argument outside data. Could you please using `disable_lora` inside `data.meta_info`. Like
```python
disable_lora = data.meta_info.pop('disable_lora', False)
adapter_ctx = self.actor.actor_module.disable_adapter() if disable_lora else nullcontext()
```